### ISSUE TITLE
requirements?

### ISSUE THREAD

**Issue by teuben** [rankinstudio/unWrap#1]
Nice job this module.  This is not an issue, just comment:

 I got a bit confused why (in the ubuntu?) example there was a mix between using apt and pip.  moreover, i found out that the name in pip for opencv was reversed, ie I had to do

     pip install opencv-python

to take away all confusion, perhaps add a requirements.txt file, but then you would have to supply a setup.py file I guess.

**Comment by rankinstudio** [comment]
Yeah, sorry. I am working on ubuntu. Just find a way to get those installed I guess lol.

Enjoy
